common/src/main/java/us/myles/ViaVersion/api/protocol/VersionRange.java

```diff
@@ -17,7 +17,7 @@
     public VersionRange(String baseVersion, int rangeFrom, int rangeTo) {
         Preconditions.checkNotNull(baseVersion);
         Preconditions.checkArgument(rangeFrom >= 0);
-        Preconditions.checkArgument(rangeTo > rangeFrom && rangeTo > 0);
```

Comment:
Why?

Reply:
That's correct, I have that locally commited as well